dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowVersionDAO.java

```diff
@@ -55,4 +93,43 @@ public List<WorkflowVersion> getTagsByWorkflowIdOrderedByLastModified(long workf
         query.setMaxResults(limit);
         return query.getResultList();
     }
+
+    private List<Predicate> processQuery(String sortCol, String sortOrder, CriteriaBuilder cb, CriteriaQuery query, Root<WorkflowVersion> version) {
+        List<Predicate> predicates = new ArrayList<>();
+
+        if (!Strings.isNullOrEmpty(sortCol)) {
+            Path<Object> sortPath;
+            if ("open".equalsIgnoreCase(sortCol)) {
+                sortPath = version.get("versionMetadata").get("publicAccessibleTestParameterFile");
```

Comment:
You could create a variable for `version.get("versionMetdata")` to reduce duplication